awx/main/access.py

```diff
@@ -1332,11 +1332,13 @@ def filtered_queryset(self):
     @check_superuser
     def can_add(self, data):
         if not data:  # So the browseable API will work
-            return Organization.accessible_objects(self.user, 'execution_environment_admin_role').exists()
+            return Organization.accessible_objects(self.user, 'member_role').exists()
         return self.check_related('organization', Organization, data)
 
     @check_superuser
     def can_change(self, obj, data):
+        if self.user.is_superuser:
+            return True
         if obj and obj.organization_id is None:
             raise PermissionDenied
         if self.user not in obj.organization.execution_environment_admin_role:
```

Comment:
I suspect that this needs to check that the user is not in `obj.organization.admin_role` as well.